diesel/src/macros/mod.rs

```diff
@@ -194,6 +198,24 @@ macro_rules! __diesel_column {
 /// # fn main() {}
 /// ```
 ///
+///
+/// If you want to add documentation for the generated code you could use the
+/// following syntax:
+///
+/// ```
+/// #[macro_use] extern crate diesel;
+///
+/// table! {
+///
+///     /// Table documentation
+///     posts {
+///         id -> Integer => "Id column documentation",
+///         title -> Text => "Title column documentation",
```

Comment:
In theory this could work, but when I tried it it resulted in parsing ambiguous.